repo/remote.go

```diff
@@ -419,7 +423,6 @@ func defaultModInfo(rc *RemoteCache, importPath string) (modPath string, err err
 
 	goTool := findGoTool()
 	env := append(os.Environ(), "GO111MODULE=on")
-
```

Comment:
same here, seems to be a no-ops style improvement that could be a separate PR

Reply:
fixed